OpenRA.Mods.Common/Traits/World/Pathfinder/Constants.cs

```diff
@@ -0,0 +1,21 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public static class Constants
+	{
+		public const double Sqrt2 = 1.414;
```

Comment:
We can't rely on floating point math to be deterministic across all runtimes.  The cases that use this will need to be changed to use integer math.